metricbeat/module/apache/status/_meta/testdata/docs.plain-expected.json

```diff
@@ -19,7 +19,7 @@
                     "system": 0.2,
                     "user": 0.16
                 },
-                "hostname": "127.0.0.1:34545",
+                "hostname": "127.0.0.1:36557",
```

Comment:
why does it modify this line?

Reply:
It happens every time this file is regenerated.

Reply:
:-(